tests/test_wave_03.py

```diff
@@ -3,24 +3,45 @@
 from adagrams.game import score_word
 
 def test_score_word_accurate():
+    #Arrange
+    words = ["A","DOG","WHIMSY"]
+    #Act
+    result1 = score_word(words[0])
+    result2 = score_word(words[1])
+    result3 = score_word(words[2])
     # Assert
-    assert score_word("A") == 1
-    assert score_word("DOG") == 5
-    assert score_word("WHIMSY") == 17
+    assert result1 == 1
+    assert result2 == 5
+    assert result3 == 17
 
 def test_score_word_accurate_ignores_case():
+    #Arrange
+    words = ["a","dog","wHiMsY"]
+    #Act
+    result1 = score_word(words[0])
+    result2 = score_word(words[1])
+    result3 = score_word(words[2])
     # Assert
-    assert score_word("a") == 1
-    assert score_word("dog") == 5
-    assert score_word("wHiMsY") == 17
+    assert result1 == 1
+    assert result2 == 5
+    assert result3 == 17
```

Comment:
It's ok to have use the return value of an expression in an assertion statement, like:
```
assert score_word("a") == 1
```

We don't need to first store the return value in a variable and then use the variable in your assertion statements. 

However, there may be times that you do find that having descriptively named variables pointing to the return value of an expression is what you want in a test. For this test case, I think it's ok just to have:
```
assert score_word("whatever your argument is") == some int